Bottom: 6e7bf14bc462d29b0d1f10e7acf354b93e7bed4f
Top:    83401534249699b3aaa0dce2863db1dbe3c25ea2
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-18 13:35:48 +0800

Refresh of introduce-skb_ring

---

diff --git a/include/linux/skb_ring.h b/include/linux/skb_ring.h
index 85d9c03e..caa759b 100644
--- a/include/linux/skb_ring.h
+++ b/include/linux/skb_ring.h
@@ -1,9 +1,5 @@
 /*
- *	Definitions for the 'struct sk_buff' memory handlers.
- *
- *	Authors:
- *		Alan Cox, <gw4pts@gw4pts.ampr.org>
- *		Florian La Roche, <rzsfl@rz.uni-sb.de>
+ *	skb ring implemented with circular buffer.
  *
  *	This program is free software; you can redistribute it and/or
  *	modify it under the terms of the GNU General Public License
